src/Umbraco.Forms.Integrations.Automation.Zapier/Services/UserValidationService.cs

```diff
@@ -53,14 +92,38 @@ public async Task<bool> Validate(string username, string password, string apiKey
 
             if (!isUserValid) return false;
 
-            if (!string.IsNullOrEmpty(_zapierSettings.UserGroupAlias))
+            if (!string.IsNullOrEmpty(_zapierFormsSettings.UserGroupAlias))
```

Comment:
This using the "forms" version only - is that right?

Reply:
Correct.